content/pricing/free.textile

```diff
@@ -55,6 +55,12 @@ To downgrade your account to a Free package:
 2. Log in to your "account":https://ably.com/login and select *Billing* from the *Account* menu.
 3. Click the *Downgrade* button.
 
-If you downgrade to a Free package, you will finish the month on your current package with its associated limits and benefits. At the beginning of the following month you will receive a final invoice covering your last month's consumption and package fee. Once the final invoice has been paid, your payment details are removed from the system.
+If you downgrade from a Standard or Pro package, you will finish the month on the Standard package with its associated limits and benefits. At the beginning of the following month you will receive a final invoice covering your last month's consumption. Once the final invoice has been paid, your payment details are removed from the system.
 
-If you wish to cancel your account, have the account owner "contact us":https://ably.com/support. Be aware that this will permanently delete all the information associated with your account.
+<aside data-type='note'>
+<p>Be aware that this will permanently delete all the information associated with your account.</p>
+</aside>
+
+h2(#close). Close your account
+
+You can "close your account":/docs/account/users#close at any time if you are on the Free package.
```

Comment:
Should we re-word this to be 'by first downgrading to the a Free package' or similar?